packages/node/src/node/ServerNode.ts

```diff
@@ -180,13 +206,104 @@ export class ServerNode<T extends ServerNode.RootEndpoint = ServerNode.RootEndpo
     protected override async initialize() {
         await ServerEnvironment.initialize(this);
 
+        this.env.get(FabricManager).events.deleted.on(fabric => this.sanitizeFabricData(fabric));
```

Comment:
I don't think there's any reason this logic needs to be in ServerNode. Could we move to a standalone component?

I would create a file that exports a "handleDeletedFabrics" or something. Maybe in behaviors/operational-credentials.  Then install it somewhere central like ServerNodeEnvironment

Reply:
Ok I adjusted it ... please have another look